app/src/main/kotlin/com/simplemobiletools/filemanager/pro/activities/ReadTextActivity.kt

```diff
@@ -83,15 +87,30 @@ class ReadTextActivity : SimpleActivity() {
         super.onActivityResult(requestCode, resultCode, resultData)
         if (requestCode == SELECT_SAVE_FILE_INTENT && resultCode == Activity.RESULT_OK && resultData != null && resultData.data != null) {
             val outputStream = contentResolver.openOutputStream(resultData.data!!)
-            saveTextContent(outputStream)
+            saveTextContent(outputStream, shouldExitAfterSaving = requestCode == SELECT_SAVE_FILE_AND_EXIT_INTENT,
+                shouldOverwriteOriginalText = getRealPathFromURI(intent.data!!) == filePath)
```

Comment:
this is way too complicated for 1 line, split it please

Reply:
Sure thing, I just tried to emulate the project's code style rather than writing it the way I prefer. Namely, when it comes to lines length, arguments list, named arguments, etc.

Reply:
it isnt about the length itself but actual logical things as arguments

Reply:
Sure, that makes sense. But I usually prefer not to declare variables that are only used once as an argument. Instead, I usually use named arguments on separate lines like this:
```
val selectedFilePath = getRealPathFromURI(intent.data!!)

saveTextContent(
	outputStream = outputStream,
	shouldExitAfterSaving = requestCode == SELECT_SAVE_FILE_AND_EXIT_INTENT,
	shouldOverwriteOriginalText = selectedFilePath == filePath
)
```

Reply:
avoid doing that here, it is too complicated and can lead to bugs

Reply:
Gotcha, no problems 👍